test/cpp/jit/test_gpu.cpp

```diff
@@ -8286,6 +8286,246 @@ TEST(NVFuserTest, FusionNonUniqueBroadcastSize_CUDA) {
   ASSERT_ANY_THROW(tv3->computeAt(tv4, -1));
 }
 
+TEST(NVFuserTest, FusionBiasGeluFwd_CUDA) {
+  Fusion fusion;
+  FusionGuard fg(&fusion);
+
+  const float k_079 = 0.79788456;
+  const float k_004 = 0.044715;
+
+  std::vector<TensorView*> tvs;
```

Comment:
what is the reason for using the tvs vector instead of naming each intermediate value? (which would avoid the need for the `// Tn` comments as well)

Reply:
At some point, I thought that would look nicer, but it doesn't seem so now. I'll simplify this part.